tests/BranchTypeConfigurationWrapperTest.php

```diff
@@ -29,38 +31,55 @@ public function setUp(): void
     }
 
     /**
-     * @return BranchTypeConfigurationAKVWrapper[][]|BranchTypeConfigurationKMSWrapper[][]
+     * @return BranchTypeConfigurationAKVWrapper[][]|BranchTypeConfigurationKMSWrapper[][]|BranchTypeConfigurationGKMSWrapper[][]
      */
     public function wrapperProvider(): array
     {
+        putenv('GOOGLE_APPLICATION_CREDENTIALS=' . getenv('TEST_GOOGLE_APPLICATION_CREDENTIALS'));
+
         $kmsOptions = new EncryptorOptions(
             stackId: 'some-stack',
             kmsKeyId: self::getKmsKeyId(),
             kmsRegion: self::getKmsRegion(),
             backoffMaxTries: 1,
         );
-        $branchTypeWrapperKMS = new BranchTypeConfigurationKMSWrapper(
+        $gkmsOptions = new EncryptorOptions(
+            stackId: 'some-stack',
+            gkmsKeyId: self::getGkmsKeyId(),
+            backoffMaxTries: 1,
+        );
```

Comment:
Tady me to trochu zmatlo.. koukam, kde se pro `BranchTypeConfigurationKMSWrapper` pouziva `$gkmsOptions` a ono je to az dole, pro `BranchTypeConfigurationGKMSWrapper`. Klidne bych to presunul az k tomu.

Reply:
No spíš bych chtěl nejdřív udělat ty options (a případně to vytáhnout to nějakýho helperu, protože jsou vždycky stejný) a pak udělat ty wrappery (protože ty jsou v každým testu jiný). 
Nicméně, nechtěl jsem to celý ještě udělat, protože se bude ještě měnit konstruktor AKV wrapperů https://keboola.atlassian.net/browse/PST-1024

Reply:
https://keboola.atlassian.net/browse/PST-1025